packages/rke2-multus/charts/values.yaml

```diff
@@ -18,10 +18,14 @@
 
 #replicaCount: 1
 
-image:
-  repository: rancher/hardened-multus-cni
-  tag: v4.2.1-build20250607
+images:
   pullPolicy: IfNotPresent
+  thin:
+    repository: rancher/hardened-multus-cni
+    tag: v4.2.1-build20250627
+  thick:
+    repository: rancher/hardened-multus-thick
+    tag: v4.2.1-build20250627
```

Comment:
is there any possibility of building both of these into the same image and just running a different command? or is that too difficult?